types/engine.io/index.d.ts

```diff
@@ -82,7 +82,7 @@ declare namespace engine {
          * conform to the ws interface (see ws module api docs). Default value is ws.
          * An alternative c++ addon is also available by installing uws module.
          */
-        wsEngine?: "ws"|"uws";
+        wsEngine?: "ws"|"uws"|string;
```

Comment:
Can you remove this change? I do not see this necessary and on top of that it blocks us from quick fixing the issue.

Reply:
Fixed.

The fix in the original issue was `string`, so I followed it, but when I checked the code in engine.io, it certainly accepted only `ws`, `uws`. Thank you.

https://github.com/socketio/engine.io/blob/3.4.1/lib/server.js#L109

I wrote the code in the past but forgot it :)